circle.yml

```diff
@@ -1,3 +1,19 @@
 machine:
+  post:
+    - sudo apt-get install build-essential
+    - sudo apt-get install libatlas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libfreetype6-dev libpng-dev
   python:
     version: 3.5.1
+
+dependencies:
+  pre:
+    - pip install --upgrade pip
+    - pip install Cython --install-option="--no-cython-compile"
+    - pip install numpy==1.11.1 
+    - pip install scipy==0.17.1
+    - pip install pytest==3.0.4
+
+test:
+  override:
+    - py.test -v
```

Comment:
What do you think about `--verbose` rather than `-v`? I like the long names for readability.